packages/Kokkos/src/DTK_LinearBVH.cpp

```diff
@@ -0,0 +1,69 @@
+#include <details/DTK_DetailsAlgorithms.hpp>
+#include <details/DTK_DetailsTreeConstruction.hpp>
+#include <details/DTK_DetailsTreeTraversal.hpp>
+
+#include <DTK_LinearBVH.hpp>
+
+#include <Kokkos_ArithTraits.hpp>
+
+namespace DataTransferKit
+{
+
+BVH::BVH( AABB const *bounding_boxes, int n )
+    : _leaf_nodes( "leaf_nodes", n )
+    , _internal_nodes( "internal_nodes", n - 1 )
+    , _indices( "sorted_indices", n )
```

Comment:
No I was actually thinking I'd take a `Kokkos::View<DTK::Box *>` as parameter in the constructor and get the size `n` from `.extent(0)`.